tests/translator/input/api_with_fail_on_warnings.yaml

```diff
@@ -0,0 +1,22 @@
+Resources:
```

Comment:
Please specify this new test case into `tests/translator/test_translator.py`. Otherwise, the test runner won't pick up this test case

Reply:
Thanks for the feedback! @hawflau I updated `tests/translator/test_translator.py`, hopefully it looks correct!

Reply:
Okay so will I just revert the last change? @jfuss

Reply:
I didn't notice that. I thought only the error files were scanned. 

@nialdaly yep, please revert the last change. Thanks!

Reply:
Done!